java-lib/src/main/java/com/wavefront/data/Validation.java

```diff
@@ -47,9 +48,11 @@
 
   private final static LoadingCache<String, Counter> ERROR_COUNTERS = Caffeine.newBuilder().
       build(x -> Metrics.newCounter(new MetricName("point", "", x)));
+  private final static LoadingCache<String, Counter> LOG_ERROR_COUNTERS = Caffeine.newBuilder().
+          build(x -> Metrics.newCounter(new MetricName("log", "", x)));
   @SuppressWarnings("UnstableApiUsage")
   private static final RateLimiter blockedLoggingRateLimiter = RateLimiter.create(1);
-  private static final Logger log = Logger.getLogger(Validation.class.getCanonicalName());
+  private static final Logger logger = Logger.getLogger(Validation.class.getCanonicalName());
```

Comment:
why?

Reply:
because i want to use the variable name log :)) and this globally conflicts with that :)